src/js/components/covid19/budgetCategories/BudgetCategories.jsx

```diff
@@ -86,27 +79,18 @@ const BudgetCategories = () => {
                     setCount(res.data.count);
                 });
         }
-
-        const overviewRequest = fetchOverview();
-        overviewRequest.promise.then((res) => {
-            const newOverview = Object.create(BaseOverview);
-            newOverview.populate(res.data);
-            setTotalBudgetaryResources(newOverview._totalBudgetAuthority);
-            setTotalObligations(newOverview._totalObligations);
-            setTotalOutlays(Math.abs(newOverview._totalOutlays));
-        });
     }, [activeTab, defCodes]);
 
     const amounts = {
         count,
-        totalBudgetaryResources,
-        totalObligations,
-        totalOutlays
+        totalBudgetaryResources: overview._totalBudgetAuthority,
+        totalObligations: overview._totalObligations,
+        totalOutlays: overview._totalOutlays
     };
 
     return (
         <div className="body__content budget-categories">
-            <DateNote dateString={dateString} />
+            <DateNote dateString={latestSubmissionDate} />
```

Comment:
We should not need this, the DateNote should be retrieving the latest submission date from redux. All we need is `<DateNote  />`